app/views/dashboards/_actions_mobile.html.erb

```diff
@@ -8,18 +8,19 @@
     <option value="/dashboard/following_podcasts" <%= "selected" if params[:action] == "following_podcasts" %>>Following podcasts (<%= @user.following_podcasts_count %>)</option>
 
     <option value="/listings/dashboard">Listings</option>
-    <% if @user.has_role? :pro %>
-      <option value="/dashboard/pro">Pro Analytics</option>
-    <% end %>
 
-    <% if @organizations && (params[:which].blank? || params[:which] == "organization") %>
-      <% @organizations.each do |org| %>
-        <option value="/dashboard/pro/org/<%= org.id %>">Pro Analytics for <%= org.name %></option>
+    <% if @current_user_pro %>
+      <option value="<%= dashboard_pro_path %>">Pro Analytics</option>
+
+      <% if @organizations && (params[:which].blank? || params[:which] == "organization") %>
+        <% @organizations.each do |org| %>
+          <option value="<%= dashboard_pro_org_path(org.id) %>">Pro Analytics for <%= org.name %></option>
+        <% end %>
       <% end %>
     <% end %>
 
-    <% if current_user.created_at < 2.weeks.ago %>
-      <option value="/videos/new">Upload a video</option>
+    <% if policy(:video).new? %>
```

Comment:
this way we have the condition only in one place, `VideoPolicy`

Reply:
Good catch!